custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -6,15 +6,19 @@ primary_entity:
   entity: vacuum
   dps:
     - id: 101
-      name: power # clean_switch
+      name: start # clean_switch
       type: boolean
```

Comment:
I think for the value_redirect to work, a mapping is needed from the string value of the command to a boolean true:
```
mapping:
   - dps_val: true
     value: start
```
and likewise for pause.

Reply:
For the record, it works as-is. 

Without these entries in the 'command' mapping, the glance card doesn't have start/pause buttons. With these entries, the buttons appear, and work as expected. Happy to test & adjust if you recommend.